syntax_and_semantics/methods_and_instance_variables.md

```diff
@@ -7,13 +7,25 @@ class Person
   def initialize(@name : String)
     @age = 0
   end
+  
+  def age
```

Comment:
I think the original commit is good. The phrase that you quote appears to have been removed by https://github.com/crystal-lang/crystal-book/commit/8b493993f435c9e04658783751faa01f5b5c29c4